microcloud/test/includes/microcloud.sh

```diff
@@ -968,6 +968,9 @@ setup_system() {
 
   # Create a snapshot so we can restore to this point.
   if [ "${SNAPSHOT_RESTORE}" = 1 ]; then
+    echo "::warning:: ==> XXX: working around LXD shutdown bug"
```

Comment:
Please an you elaborate on what this is and reference a GH issue or PR tracking it?

Reply:
I added a link to the PR in question: https://github.com/canonical/microcloud/pull/286